stubs/geoip2/@tests/stubtest_allowlist.txt

```diff
@@ -1,6 +1,5 @@
 geoip2.database.Reader.__exit__
 geoip2.database.Reader.__init__
-geoip2.errors.HTTPError.__init__
```

Comment:
I don't really understand why this must be removed when changing `METADATA.toml`, but CI is green.

Reply:
I believe it's because the tests were using the latest version of the package (4.x) before I changed it to 3.0 and the allowlist was probably necessary for that. But since it's unnecessary for 3.x, the tests failed.